tests/ext/sandbox-regression/handle_many_hooks.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+The tracer should not crash when many hooks are installed
```

Comment:
Out of curiosity, did this test failed consistently before fixing?

Reply:
Yes.